### PR TITLE
chore: fix dependabot config

### PR DESCRIPTION
**Pull request by JoergSiebahn**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -15,25 +15,27 @@ updates:
   labels:
     - 'dependencies'
 # maintaining v2
-- package_manager: "java:gradle"
-  target_branch: "release/2.x.x"
+- package-ecosystem: "java:gradle"
```
Comment (christopher-cudennec) [inline .github/dependabot.yml:18]: Is "java:gradle" the same like "gradle" used above?

**Review by elbiocaetano — APPROVED** [review]

File: .github/dependabot.yml [hunk 2]
```diff
@@ -15,25 +15,27 @@ updates:
...
-  commit_message:
+  commit-message:
     prefix: "fix"
-    prefix_development: "chore"
-    include_scope: true
-- package_manager: "github_actions"
-  target_branch: "release/2.x.x"
+    prefix-development: "chore"
+    include: scope
+- package-ecosystem: "github-actions"
```
Comment (christopher-cudennec) [inline .github/dependabot.yml:30]: Oh... did that work? Do we want updates for Github actions only for the release branch?

Comment (JoergSiebahn) [reply .github/dependabot.yml:30]: That's covered with renovate for the main branch to group the updates: [Example](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2020)

File: .github/dependabot.yml (hunk 1 above)
Comment (JoergSiebahn) [reply .github/dependabot.yml]: [fixed](https://github.com/SDA-SE/sda-dropwizard-commons/compare/9ae8149ac9b36d06162a5c16984f982d7c1d4ea4..cfc18b53e50f76cf31e29dda690120e8440ab273)

**Review by christopher-cudennec — APPROVED** [review]

**Pull request merged**